ui/src/report/services/payment.service.ts

```diff
@@ -9,15 +9,11 @@ import { PaymentChartResponse } from '../types/api/payment-chart-response';
 
 @Injectable('payment')
```

Comment:
`payment` doesn't make any sense anymore. Change this to `report.service.ts` instead